bootstrap.yml

```diff
@@ -118,13 +146,39 @@ sources:
     tag: 'releases/gcc-11.1.0'
     version: '11.1.0'
     tools_required:
+      # TODO Dennis: autoconf 2.71 is too new, I'll take it during the toolchain upgrade.
       - host-autoconf-v2.69
-      - host-automake-v1.11
+      - host-automake-v1.16
+      - host-libtool
     regenerate:
-      - args: ['autoconf']
+      - args: ['autoreconf', '-fvi', '-I../config']
+        workdir: '@THIS_SOURCE_DIR@/gcc'
+      - args: ['autoreconf', '-fvi', '-I../config']
+        workdir: '@THIS_SOURCE_DIR@/libstdc++-v3'
+      - args: ['libtoolize', '-cfvi']
         workdir: '@THIS_SOURCE_DIR@/gcc'
-      - args: ['autoconf']
+      - args: ['libtoolize', '-cfvi']
         workdir: '@THIS_SOURCE_DIR@/libstdc++-v3'
+      - args: ['autoreconf', '-fvi', '-I../config']
+        workdir: '@THIS_SOURCE_DIR@/'
+      - args: ['libtoolize', '-cfvi']
+        workdir: '@THIS_SOURCE_DIR@/'
+      - args: ['autoreconf', '-fvi', '-I../config']
+        workdir: '@THIS_SOURCE_DIR@/libbacktrace'
+      - args: ['libtoolize', '-cfvi']
+        workdir: '@THIS_SOURCE_DIR@/libbacktrace'
+      - args: ['autoreconf', '-fvi', '-I../config']
+        workdir: '@THIS_SOURCE_DIR@/lto-plugin'
+      - args: ['libtoolize', '-cfvi']
+        workdir: '@THIS_SOURCE_DIR@/lto-plugin'
+      - args: ['autoreconf', '-fvi', '-I../config']
+        workdir: '@THIS_SOURCE_DIR@/libcc1'
+      - args: ['libtoolize', '-cfvi']
+        workdir: '@THIS_SOURCE_DIR@/libcc1'
+      - args: ['autoreconf', '-fvi', '-I../config']
```

Comment:
Why repeat this instead of just overriding the config.sub autoconf holds?

Reply:
According to mint, this is the better way of doing it. I tried not doing it in all directories but that will give libtool version mismatches.

Reply:
what's `-I../config` for? my guess was to specify where config.{guess,sub} are, but we should just put them into whatever provides them instead

Reply:
also autoreconf is broken on GCC, it has a script that can update files in the right order, but since we know what we patch, it's probably better to just invoke the tools directly in the specific case of GCC

Reply:
yes, by coincidence, not design. we should definitely prefer to have no `-I` since that can only confuse whoever else is trying to use autoconf